shell/browser/common_web_contents_delegate.cc

```diff
@@ -184,6 +189,24 @@ CommonWebContentsDelegate::CommonWebContentsDelegate()
 
 CommonWebContentsDelegate::~CommonWebContentsDelegate() = default;
 
+#if BUILDFLAG(ENABLE_PDF_VIEWER)
+class ElectronPDFWebContentsHelperClient
```

Comment:
Can you move this to separate file under `shell/browser/pdf` ?

Reply:
There's no `pdf/` directory currently, do you think this warrants a new dir?

Reply:
Yeah its better, future implementations can go in there, avoids polluting this file.